stable/influxdb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: influxdb
-version: 3.0.2
+version: 3.0.3
```

Comment:
The new feature was backward-compatible here. So, please bump the middle version, like`3.1.0`.